### PR TITLE
Add related records document

### PR DESCRIPTION
**Pull request by akong-sf**
- Added a document to introduce the related records feature.
- Added comment about GraphQL prefetching issue.

### CODE REVIEW

File: docs/RelatedRecords.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Related Records
+
+[The related records starter kit example](../force-app/main/default/lwc/accountRelatedContacts) demonstrates the capability to view related records e.g. `Contacts` from an object e.g `Account` using GraphQL queries. When the device is online and while briefcase priming, the related records are prefetched and stored in the cache to be able to view in offline mode. There is currently a well known issue that GraphQL query prefetch fails, and as a result, accountRelatedContacts LWC will not work offline. An expected fix will come in November 2023.
```
Comment (Taost) [inline docs/RelatedRecords.md:3]: Could you add the KI link for the GraphQL issue?

File: docs/RelatedRecords.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          }
+        }
+      }
+    `;
+  }
+```
+
+See the [Query Objects Examples documentation](https://developer.salesforce.com/docs/platform/graphql/guide/query-record-examples.html) for more details on how to use GraphQL queries.
+
+See the [Query Objects Examples developer document for more details on how to use GraphQL queries.
```
Comment (Taost) [inline docs/RelatedRecords.md:46]: Duplicated?

File: docs/RelatedRecords.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+Depending on your use case, you can add the `accountRelatedContacts` LWC as a quick action by creating a new file within the `quickActions` folder that includes the following
```
Comment (Taost) [inline docs/RelatedRecords.md:46]:
```suggestion
Depending on your use case, you can deploy the `accountRelatedContacts` LWC as a quick action by creating a new file within the `quickActions` folder that includes the following
```

File: docs/RelatedRecords.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
+# Related Records
+
+[The related records starter kit example](../force-app/main/default/lwc/accountRelatedContacts) demonstrates the capability to view related records e.g. `Contacts` from an object e.g `Account` using GraphQL queries. When the device is online and while briefcase priming, the related records are prefetched and stored in the cache to be able to view in offline mode. There is currently a well known issue that [GraphQL query prefetch fails](https://issues.salesforce.com/issue/a028c00000xGGwEAAW/graphql-query-fails-prefetch-with-an-unknown-field-warning), and as a result, accountRelatedContacts LWC will not work offline. An expected fix will come in November 2023 and there is a workaround that can be implemented in this [Knowledge Article](https://help.salesforce.com/s/articleView?language=en_US&id=000396405&type=1).
```
Comment (AdamLiechty) [inline docs/RelatedRecords.md:3]:
```suggestion
[The related records starter kit example](../force-app/main/default/lwc/accountRelatedContacts) demonstrates the capability to view related records, e.g. `Contacts`, from an object, e.g `Account`, using GraphQL queries. When the device is online and while a user's briefcase is priming, the related records are prefetched and stored in the cache to be able to view in offline mode. There is currently a well known issue that [GraphQL query prefetches fail](https://issues.salesforce.com/issue/a028c00000xGGwEAAW/graphql-query-fails-prefetch-with-an-unknown-field-warning), and as a result, the `accountRelatedContacts` LWC will not work offline. A fix is expected in November 2023, and there is a workaround that can be implemented in this [Knowledge Article](https://help.salesforce.com/s/articleView?language=en_US&id=000396405&type=1).
```

File: docs/RelatedRecords.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+In this example, we performed a GraphQL query to fetch related contacts from an account record.
```
Comment (AdamLiechty) [inline docs/RelatedRecords.md:5]:
```suggestion
In this example, we perform a GraphQL query to fetch related contacts from an account record.
```

File: docs/RelatedRecords.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+          }
+        }
+      }
+    `;
+  }
+```
+
+See the [Query Objects Examples documentation](https://developer.salesforce.com/docs/platform/graphql/guide/query-record-examples.html) for more details on how to use GraphQL queries.
+
+Depending on your use case, you can deploy the `accountRelatedContacts` LWC as a quick action by creating a new file within the `quickActions` folder that includes the following
```
Comment (AdamLiechty) [inline docs/RelatedRecords.md:46]:
```suggestion
Depending on your use case, you can deploy the `accountRelatedContacts` LWC as a quick action by creating a new file within the `quickActions` folder that includes the following:
```

File: docs/RelatedRecords.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+</QuickAction>
+```
+
+## How to Configure
+
+1. [Configure your Offline Briefcase](../README.md#define-an-offline-briefcase) with the `Account` sObject.
+2. Deploy the following LWCs to the org, which can also be added as quick actions to the sObjects:
+    - `viewAccountRecord` (`Account.view`) and `viewContactRecord` (`Contact.view`)
+    - `accountRelatedContacts`
+        - If the accountRelatedContacts LWC is added as a quick action, you will have to update the page layout to include it.
```
Comment (AdamLiechty) [inline docs/RelatedRecords.md:65]:
```suggestion
        - If the `accountRelatedContacts` LWC is added as a quick action, you will have to update the page layout to include it.
```

**Review by AdamLiechty — CHANGES_REQUESTED** [review]

File: docs/RelatedRecords.md (hunk 4 above)
Comment (sfdctaka) [reply docs/RelatedRecords.md:3]: Should we be more discrete about November 2023 date?

**Review by AdamLiechty — APPROVED** [review]

**Review by Taost — APPROVED** [review]

File: docs/RelatedRecords.md (hunk 4 above)
Comment (akong-sf) [reply docs/RelatedRecords.md]: I think it should be fine as long as we mention the issue. We do plan to remove this once the issue is fixed

Comment (sfdctaka) [reply docs/RelatedRecords.md]: What if it is not fixed in November but December. Or even next year?

Comment (akong-sf) [reply docs/RelatedRecords.md]: Good point. We should change the wording to "the near future".

**Pull request merged**